tests/cases/fourslash/completionsImport_weirdDefaultSynthesis.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: /collection.ts
+//// class Collection {
+////   public static readonly default: typeof Collection = Collection;
+//// }
+//// export = Collection as typeof Collection & { default: typeof Collection };
+
+// @Filename: /index.ts
+//// Colle/**/
+
+verify.applyCodeActionFromCompletion("", {
+  name: "Collection",
+  source: "/collection",
+  description: `Import 'Collection' from module "./collection"`,
+  preferences: {
+    includeCompletionsForModuleExports: true
+  },
+  newFileContent: `import Collection = require("./collection");\n\nColle`
```

Comment:
Possibly I'm misreading something, but shouldn't "Colle" be completed to "Collection"?

Reply:
`applyCodeActionFromCompletion` only does what it says; it doesn’t apply the completion itself.